src/common/components/CoreLayout/Sidebar/SidebarItem.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+import { type SidebarListItem } from "./types";
+
+export const SidebarItem = ({
```

Comment:
Lets move `SidebarItem.tsx` into `/common/components/SidebarItem`, and create a `index.ts` to `export * from './SidebarItem`.

This way, it's easier to find the component at a glance from looking at the file directory